reports/report_builder.py

```diff
@@ -4600,7 +4613,13 @@ def tabulate_historical(self, ws, current_ws, cols, rows, c=None, r=6, write_row
                         value = data[canon(col_heading)].get(canon(row_heading))
 
                         if value is None:
-                            value = ['n/a'] * values_per_col
+                            # check if it's due to having % and N
+                            p = data.get(canon(col_heading), {}).get('%', {})
+                            n = data.get(canon(col_heading), {}).get('n', {})
+                            if (p or n):
+                                value = [p.get(canon(row_heading), "n/a"), n.get(canon(row_heading), "n/a")]
+                            else:
+                                value = ['n/a'] * values_per_col
```

Comment:
Can you explain a bit how this works / why wasn't it done before?

Reply:
From what I've seen, in 2015, we didn't have any historical data that contained 2 values i.e `%` and `n`. Most of them(2010, 2005 data) were just in the following format
```
{'africa': 0.44, 'asia': 0.49, 'caribbean': 0.34, 'europe': 0.64, 'latin america': 0.44, 'middle east': 0.34, 'north america': 0.83, 'pacific islands': 0.43}
```
In 2015, data containing `%` and `n` was added and that is what we now need to extract.
Basically, this part of the code helps extract values that are one level down 
`{'internet': {'%': {'celebrity, arts and media, sports': 0.182608695652174, ...}, 'n': {'celebrity, arts and media, sports': 84, ...}}, `